test/unit/Sql/Ddl/AlterTableTest.php

```diff
@@ -88,14 +98,16 @@ public function testGetSqlString()
         $at->changeColumn('name', new Column\Varchar('new_name', 50));
         $at->dropColumn('foo');
         $at->addConstraint(new Constraint\ForeignKey('my_fk', 'other_id', 'other_table', 'id', 'CASCADE', 'CASCADE'));
-        $at->dropConstraint('my_index');
+        $at->dropConstraint('my_constraint');
+        $at->dropIndex('my_index');
         $expected = <<<EOS
 ALTER TABLE "foo"
  ADD COLUMN "another" VARCHAR(255) NOT NULL,
  CHANGE COLUMN "name" "new_name" VARCHAR(50) NOT NULL,
  DROP COLUMN "foo",
  ADD CONSTRAINT "my_fk" FOREIGN KEY ("other_id") REFERENCES "other_table" ("id") ON DELETE CASCADE ON UPDATE CASCADE,
- DROP CONSTRAINT "my_index"
+ DROP CONSTRAINT "my_constraint"
```

Comment:
I will have to get to this later this week, but did you have any trouble when you ran the tests? I may be off-base, but a trailing comma may be missing here.

Reply:
I had missed it yes, however added a followup commit to fix that issue.